hatch-dependency-coversion/tests/utils.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from os import linesep
+import subprocess
+
+
+def _indent(msg: str, count: int = 0) -> str:
+    lines = msg.split(linesep)
+    indent = "\t" * count
+    return indent + f"{indent}{linesep}".join(lines)
+
+
+class SubprocFailed(Exception):
+    def __init__(self, command: str, returncode: int, stdout: str, stderr: str) -> None:
+        self.message = f"Command {command} failed: {returncode}!{linesep}stdout:{linesep}{_indent(stdout, 1)}{linesep}stderr:{linesep}{_indent(stderr, 1)}"
+        self.command = command
+        self.returncode = returncode
+        self.stdout = stdout
+        self.stderr = stderr
+
+    def __repr__(self) -> str:
+        return f"<SubprocFailed command={self.command} returncode={self.returncode}>"
+
+    def __str__(self) -> str:
+        return self.message
+
+
+def build_wheel(proj_path: Path, output_path: Path | None = None) -> Path:
```

Comment:
Looks like this `|` syntax is invalid on Python 3.8. :(

Reply:
Not with future annotations!